include/xtensor/xsort.hpp

```diff
@@ -525,6 +525,38 @@ namespace xt
         std::copy(sorted.begin(), end, result.begin());
         return result;
     }
+
+    /**
+     * Find the set difference of two xexpressions. This returns a flattened xtensor with
+     * Return the sorted, unique values in ar1 that are not in ar2.
```

Comment:
"Return" should be removed or the previous phrase has been truncated.

Reply:
Done